app/src/main/java/com/framgia/fsalon/data/source/remote/CustomerBookingRemoteDataSource.java

```diff
@@ -0,0 +1,55 @@
+package com.framgia.fsalon.data.source.remote;
+
+import com.framgia.fsalon.data.model.BookingCustomer;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.data.model.BookingCustomer. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports